src/main/java/bio/terra/cloudfiletodatastore/deltalayer/model/json/PointCorrectionOperation.java

```diff
@@ -0,0 +1,36 @@
+package bio.terra.cloudfiletodatastore.deltalayer.model.json;
+
+import java.util.UUID;
+
+public class PointCorrectionOperation {
+
+  private UUID datarepoRowId;
+
+  private String name;
+
+  private String value;
```

Comment:
in the json files that exist in GCS, these won't always be strings. The values here will be any json datatype, so you might see:
```
"value": "my_new_value"
```
for strings, 
```
"value": 123.45
```
for numbers,
```
"value": true
```
for booleans.